### ISSUE TITLE
Improve output format in XML, PDF and EPUB

### ISSUE THREAD

**Issue by arojascastro** [humboldtdigital/ProHD#6]
I have tested a sample of those on the server with the aim of generating files in various formats using the Download option.

Preliminary analysis

-     PDF (LaTeX version) does not work in any case.
-     Tex source code works in all cases, but the file has no metadata
-     PDF (fo version) works in some cases (in letters), but the file has no metadata and the diplomatic transcription is provided.
-     fo source code works in all cases, does not appear to be malformed.
-     EPUB works in all cases, but the generated documents do not have metadata and the diplomatic transcription is provided.
-     XML does not work in any case

Question: is this an ODD problem? @tuurma 

If so, can you help me to identify errors and fix them? This could be part of our workshop as well if requires longer time and bigger effort,

**Comment by tuurma** [comment]
For PDF generation you need to configure LaTeX on the server and point to a relevant path in modules/config.xqm

```
{
  "details" : null,
  "description" : "Cannot run program \"/usr/local/bin/pdflatex\" (in directory \"/tmp\"): error=2, No such file or directory. An IO error occurred while executing the process /usr/local/bin/pdflatex: Cannot run program \"/usr/local/bin/pdflatex\" (in directory \"/tmp\"): error=2, No such file or directory [at line 179 of unknown]"
}
```

**Comment by arojascastro** [comment]
Thank you! I have contacted Markus and shared the information.

**Comment by tuurma** [comment]
We'd need more concrete specifications for what is expected, ideally a screenshot of what is with a mockup for what should be, perhaps starting with FO (or LaTeX if you get the setup on the server right)

**Comment by arojascastro** [comment]
True. Basically, a first improvement would be to include some of the metadata in the output document that allows to identify the edition, source, and editor. I will prepare something before the workshop.
